src/app/loadout/fashion/FashionDrawer.tsx

```diff
@@ -515,28 +533,38 @@ function FashionSocket({
         socket?.plugSet?.plugs.some((p) => p.plugDef.hash === plugHash)) ||
       socket?.reusablePlugItems?.some((p) => p.plugItemHash === plugHash && p.enabled));
 
+  let content = null;
+
+  if (plug) {
```

Comment:
I didn't see anything wrong with the ternary operator this used before

Reply:
I can revert this change, I just have a slight preference for this kind of flow but happy to go with your preference.

Reply:
Done